test/printf.c

```diff
@@ -0,0 +1,20 @@
+#include <stdio.h>
```

Comment:
Ok #792 now creates `i64.store8` in asm2wasm, verified in the output. But it still executes this file ok without this PR... so I'm not seeing this test provide coverage for the issue, but no idea why. Does it depend on something else in the wasm-backend output?

Anyhow, if this is a hassle to figure out, I guess it's ok to leave it.